__tests__/app.test.js

```diff
@@ -22,6 +22,20 @@ describe("GET /api/(non existent end point)", () => {
     });
 })
 
+describe("GET /api", () => {
+    test("200 responds with an object describing all of the endpoints ", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({body}) => {
+            
+          for(const key in body) {
+   
+            expect(typeof body[key].description).toBe('string')
```

Comment:
We want to expect as a minimum that each entry describing an endpoint has the keys description, queries, and exampleResponse even if any of those fields are empty. Please refactor to do so 🙏

Alternatively, you could make this test more accurate and dynamic by requiring in the endpoints.json as you have done in your controller, and expecting that your response is equal to that object 🧠